src/js/chrome/iqeEnablement.js

```diff
@@ -73,7 +73,7 @@ export default {
     xhrResults.map((e) => console.log(`[iqe] xhr incomplete: ${e._url}`)); // eslint-disable-line no-console
     Object.values(fetchResults).map((e) => console.log(`[iqe] fetch incomplete: ${e}`)); // eslint-disable-line no-console
 
-    return xhrResults.length > 0 || fetchResults.length > 0;
+    return xhrResults.length > 0 || fetchResults.hasOwnProperty('length') ? fetchResults.length : Object.values(fetchResults).length > 0; // eslint-disable-line no-prototype-builtins
```

Comment:
You don't really have to check for the `length` property. The `Array` is object as well so you can simplify this to
```suggestion
    return xhrResults.length > 0 || Object.values(fetchResults).length > 0;
```

Reply:
It's my fault for suggesting that. I have seen this piece of code for the first time and did not know what was the original intent.

Reply:
@karelhala thanks! that sounds good to me, suggestion applied